app/main.py

```diff
@@ -2,0 +1,17 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split(" ")
+    path_2 = command_parts[2].split("/")
+    with open(command_parts[1], "r") as file_read:
```

Comment:
Opening a file without checking if it exists can lead to a `FileNotFoundError`. Consider adding a check to ensure the file exists before attempting to open it.